config/development.json

```diff
@@ -12,6 +12,6 @@
     "webSocketPort": 9222
   },
   "firefox": {
-    "webSocketPort": 9000
+    "webSocketPort": 6080
```

Comment:
oh nice. sorry, missed that. In that case I think we should have two fields
```js
firefoxProxyPort: 9000,
firfoxWebsocketPort: 6080
```

and feature flag at that location so the appropriate one is used